reflex/compiler/compiler.py

```diff
@@ -116,6 +119,54 @@ def _compile_page(
     )
 
 
+def compile_root_stylesheet(stylesheets: List[str]) -> Tuple[str, str]:
+    """Compile the root stylesheet.
+
+    Args:
+        stylesheets: The stylesheets to include in the root stylesheet.
+
+    Returns:
+        The path and code of the compiled root stylesheet.
+    """
+    output_path = utils.get_root_stylesheet_path()
+
+    code = _compile_root_stylesheet(stylesheets)
+
+    return output_path, code
+
+
+def _compile_root_stylesheet(stylesheets: List[str]) -> str:
+    """Compile the root stylesheet.
+
+    Args:
+        stylesheets: The stylesheets to include in the root stylesheet.
+
+    Returns:
+        The compiled root stylesheet.
+
+    Raises:
+        FileNotFoundError: If a specified stylesheet in assets directory does not exist.
+    """
+
+    def is_valid_url(url):
+        result = urlparse(url)
+        return all([result.scheme, result.netloc])
+
+    sheets = [constants.TAILWIND_ROOT_STYLE_PATH]
+    for stylesheet in stylesheets:
+        if not is_valid_url(stylesheet):
+            # check if stylesheet provided exists.
+            if not os.path.exists(
+                Path.cwd() / constants.APP_ASSETS_DIR / stylesheet.strip("/")
+            ):
+                raise FileNotFoundError(
+                    f"The stylesheet file {Path.cwd() / constants.APP_ASSETS_DIR / stylesheet} does not exist."
+                )
+            stylesheet = "/".join(["../public", stylesheet.strip("/")])
```

Comment:
Do we need to go back a directory to `../public` because this gets compiled into the `pages/_app.js` file? Maybe it would bee better to make this `/public/` which should get it from the app root directory (I think)

Reply:
Yeah, I just made a push to change this.